_aicspylibczi/Reader.cpp

```diff
@@ -146,6 +150,7 @@ Reader::sceneShape(int scene_index_)
   bool sceneBool(false);
   int sceneStart(0), sceneSize(0);
   tie(sceneBool, sceneStart, sceneSize) = scenesStartSize();
+  pixelType(); // this may look like it isn't used but it's setting m_pixelType
```

Comment:
is there a better name for this function?

Reply:
So the challenge is that there are a number of code paths that need pixeltype loaded. Rather than have a setPixelType I used the same pattern we are using in aicsimageio of lazy loading the value -- set it when it's called.  I could replace it with m_pixelType = getPixelType() but it seems purely stylistic. Am I missing something?

Reply:
It's true that getPixelType is a style choice but in this case it would be more explicit about what it's doing.  In fact something like getPixelTypeLazyInit would be even more explicit.  It would be consistent with "functions as verbs" but we never really explicitly said we'd do that for C++ code.